src/webhook.rs

```diff
@@ -38,10 +38,7 @@ pub async fn webhook(bot: Bot) -> mpsc::UnboundedReceiver<Result<Update, String>
         .parse()
         .expect("BIND_PORT value has to be an integer");
 
-    let teloxide_token = env::var("TELOXIDE_TOKEN").expect("TELOXIDE_TOKEN env variable missing");
-    let host = env::var("HOST").expect("HOST env variable missing");
-    let path = format!("/{}/api/v1/message", teloxide_token);
-    let url = format!("https://{}{}", host, path);
+    let url = env::var("WEBHOOK_URI").expect("Not detected WEBHOOK_URI set environment variable WEBHOOK_URI to start webhook");
```

Comment:
Change text in `expect` to `WEBHOOK_URI env variable missing` for being consistent with other vars.